Tests/SourceKitLSPTests/SwiftInterfaceTests.swift

```diff
@@ -0,0 +1,132 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2014 - 2022 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+
+import Foundation
+import LanguageServerProtocol
+import LSPTestSupport
+import SKTestSupport
+import SourceKitLSP
+import XCTest
+
+final class SwiftInterfaceTests: XCTestCase {
+
+  /// Connection and lifetime management for the service.
+  var connection: TestSourceKitServer! = nil
+
+  /// The primary interface to make requests to the SourceKitServer.
+  var sk: TestClient! = nil
+
+  var documentManager: DocumentManager! {
+    connection.server!._documentManager
+  }
+
+  override func setUp() {
+    connection = TestSourceKitServer()
+    sk = connection.client
+    _ = try! sk.sendSync(InitializeRequest(
+      processId: nil,
+      rootPath: nil,
+      rootURI: nil,
+      initializationOptions: nil,
+      capabilities: ClientCapabilities(workspace: nil,
+                                       textDocument: TextDocumentClientCapabilities(
+                                        codeAction: .init(
+                                          codeActionLiteralSupport: .init(
+                                            codeActionKind: .init(valueSet: [.quickFix])
+                                          )),
+                                        publishDiagnostics: .init(codeDescriptionSupport: true)
+                                       )),
+      trace: .off,
+      workspaceFolders: nil))
+  }
+  
+  override func tearDown() {
+    sk = nil
+    connection = nil
+  }
+  
+  func testSystemModuleInterface() throws {
+    let url = URL(fileURLWithPath: "/\(UUID())/a.swift")
+    let uri = DocumentURI(url)
+
+    sk.send(DidOpenTextDocumentNotification(textDocument: TextDocumentItem(
+      uri: uri,
+      language: .swift,
+      version: 1,
+      text: """
+      import Foundation
+      """)))
+    
+    let _resp = try sk.sendSync(DefinitionRequest(
+      textDocument: TextDocumentIdentifier(url),
+      position: Position(line: 0, utf16index: 10)))
```

Comment:
Do you think you could also define a `testLoc` for this instead of hardcoding the position in the tests?

Reply:
Unlike the other tests in that file, this particular test doesn't have a TestWorkspace, so `testLoc` is not readily available from what I can see. Rather, the code is provided via `DidOpenTextDocumentNotification`. Similarly, `LocalSwiftTests.swift` is full of `Positions` with hard-coded offsets.

Reply:
Oh, sorry. I should have read the test case more carefully.